### DISCUSSION TITLE
HTTPS

### DISCUSSION THREAD

**Discussion by spruce0** [plan-player-analytics/Plan#3030] [Q&A]
I created a certificate using Let's Encrypt and CertBot and wondered how to put it on my server. The files I get are .pem files, while the files I need for Plan are .jks files. How would I set HTTPS up with these?

**Comment by AuroraLS3** [comment]
Follow these instructions https://github.com/plan-player-analytics/Plan/wiki/SSL-Certificate-%28HTTPS%29-Set-Up#step-4-creating-pkcs12-from-certificate-files
